content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -9,28 +9,26 @@ menu:
     parent: About the project
 ---
 
-v1.9.0 [unreleased]
--------------------
+v1.9.0 [TK]
+-----------
 
 ### Features
 - Support user-defined *node labels*.
   Node labels allows cluster operators to assign arbitrary key-value pairs to meta and data nodes in a cluster.
   For instance, an operator might want to label nodes with the availability zone in which they're located.
-- Improve diagnostics for license problems.
-- Ingress metrics now work.
+- Improve diagnostics for license problems. Add license expiration date to `debug/vars` metrics.
+- Improved ingress metrics.
```

Comment:
Where is this explained in more detail?  I think folks will want a pointer to this....

Reply:
Working on adding docs for this now.

Reply:
Have added a link to new config docs for this. See also https://github.com/influxdata/docs-v2/pull/2529.